src/index.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
It looks like almost all of the code in this file is inside this one function. Clean code principals apply on the front end. Functions should be small and handle one thing. A lot of the code inside are function definitions that can be written outside of this function and then called from inside.